files/zerotier.fact.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python3
+
+import json
+import subprocess
+
+out = subprocess.Popen(['/usr/sbin/zerotier-cli', '-j', 'info'],
+                       stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
+
+stdout, stderr = out.communicate()
+
+try:
+    info = json.loads(stdout)
```

Comment:
`json.loads()` expects string. Bytes given. I will change this, and the occurrence on line 27, to `json.loads(stdout.decode())` after merging into develop branch.